packages/amplify_analytics_pinpoint/example/pubspec.yaml

```diff
@@ -6,7 +6,7 @@ description: Demonstrates how to use the amplify_analytics_pinpoint plugin.
 publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
```

Comment:
Nit: We should try to be consistent about using double vs single quotes. I *think* the convention is single based on Flutter docs around `pubspec.yaml`

Reply:
yes single is best.